AppKit/CPTextView/CPTextView.j

```diff
@@ -1848,18 +1848,18 @@ var kDelegateRespondsTo_textShouldBeginEditing
 
 + (JSObject)_wordBoundaryRegex
 {
-    return /(^[0-9][\.,])|(^.[^-\.,+#'"!§$%&/\(<\[\]>\)=?`´*\s{}\|¶])/m;
+    return new RegExp("(^[0-9][\\.,])|(^.[^-\\.,+#'\\\"!§$%&/\\(<\\[\\]>\\)=?`´*\\s{}\\|¶])", "m");
```

Comment:
i am a bit uneasy about the \\\" (triple backslash)
should't this be \" (one backslash for the JS parser, not the regex parser)?
not 100% sure as i did not test this, though.